code/objects/News.php

```diff
@@ -321,17 +321,18 @@ public function getMonthCreated(){
 		return $this->dbObject('PublishFrom')->Format('F');
 	}
 
-        /**
-         * Create a date-string based on the locale. Looks better.
-         * @return string
-         */
-        public function getPublished() {
-		i18n::get_date_format();
+	/**
+	 * Create a date-string based on the locale. Looks better.
+	 * @return string
+	 */
+	public function getPublished()
+	{
+		$format = i18n::get_date_format();
 		$locale = i18n::get_locale();
 		$date = new Zend_Date();
 		$date->set($this->PublishFrom, null, $locale);
-		return substr($date->getDate($locale),0,-9);
```

Comment:
I tested at an installation with fluent and de_DE / en_US locales.
Was totally broken in en_US before, works now for both.

Substring doesn't work if you also have AM/PM in the generated string this caused the bad english date...